docs/source/create_channel/create_channel_participation.md

```diff
@@ -388,7 +388,7 @@ export OSN_TLS_CA_ROOT_CERT=../config/organizations/ordererOrganizations/orderer
 export ADMIN_TLS_SIGN_CERT=../config/admin-client/client-tls-cert.pem
 export ADMIN_TLS_PRIVATE_KEY=../config/admin-client/client-tls-key.pem
 
-osnadmin channel join –-channel-id [CHANNEL_NAME]  --config-block [CHANNEL_CONFIG_BLOCK] -o [ORDERER_ADMIN_LISTENADDRESS] --ca-file $OSN_TLS_CA_ROOT_CERT --client-cert $ADMIN_TLS_SIGN_CERT --client-key $ADMIN_TLS_PRIVATE_KEY
+osnadmin channel join --channel-id [CHANNEL_NAME]  --config-block [CHANNEL_CONFIG_BLOCK] -o [ORDERER_ADMIN_LISTENADDRESS] --ca-file $OSN_TLS_CA_ROOT_CERT --client-cert $ADMIN_TLS_SIGN_CERT --client-key $ADMIN_TLS_PRIVATE_KEY
```

Comment:
Note: it isn't even visually clear in github that this character was different but in my editor it was visible.

Reply:
Yea it's annoying. I had to debug one of these last year when someone reported our commands weren't working. It's a copy paste thing. The first character is actually a double dash from auto completion. Open microsoft word and do two dashes then a word and press spacebar and it will merge them. I should actually add it to a linting script to check for them in doc. I replaced a bunch of them last year.